sufia-models/lib/sufia/models/stats/user_stat_importer.rb

```diff
@@ -47,6 +51,22 @@ def delay
       sleep @delay_secs
     end
 
+    def rescue_and_retry fail_message
+      retry_count = 0
+      begin
+        return yield
+      rescue Exception => e
```

Comment:
Rescue StandardError, not Exception: https://robots.thoughtbot.com/rescue-standarderror-not-exception